### ISSUE TITLE
Update development installation instructions

### ISSUE THREAD

**Issue by Roel** [DOV-Vlaanderen/pydov#255]
From the [pyOpenSci review](https://github.com/pyOpenSci/software-review/issues/19):

> There is no instructions about how to install the package for development in README file (e.g. prepare the environment: "pip install -r requirements_dev.txt", install the package in the development mode "python setup.py develop", etc)

There are bits and pieces of this information in the docs (in the [installation section](https://pydov.readthedocs.io/en/latest/installation.html#from-sources) and the [contributing section](https://pydov.readthedocs.io/en/latest/contributing.html#setting-up-your-environment) for example), but this might be hard to find and assemble.

I suggest:
- adding development installation instructions to the README
- adding the same instructions in the installation section in the docs
- add cross-reference between installation section and contributing section, maybe with a bit more info in the latter (regarding the different requirements files etc.)

**Comment by stijnvanhoey** [comment]
Basically, the dev get started boils down to...

### Clone repo

```
git clone git@github.com:DOV-Vlaanderen/pydov.git
```

### a development environment

```
conda create -n pydov python=3.7
conda activate pydov
```

or 

```
virtualenv pydov
source pydov/bin/activate | pydov\Scripts\activate
```

### Install dependencies for development

Inside the environment:

```
pip install -e .[devs]
```

I would put this in the README and docs installation section and refer to the contirbuting guidelines for more in-depth information. I'm rewriting these guidelines as well to better align.

**Comment by Roel** [comment]
Closed in #274
